products/compute/terraform.yaml

```diff
@@ -630,17 +606,12 @@ overrides: !ruby/object:Provider::ResourceOverrides
         custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
       sourceDiskEncryptionKey: !ruby/object:Provider::Terraform::PropertyOverride
         ignore_read: true
-        custom_expand: templates/terraform/custom_expand/source_disk_encryption_key_expand.go.erb
-        update_verb: :NOOP
-        update_url: ""
       sourceDiskEncryptionKey.rawKey: !ruby/object:Provider::Terraform::PropertyOverride
```

Comment:
Source disk encryption should also have `sha256`.

Reply:
Sadly, nope: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/api.yaml#L3177